### PR TITLE
Remove Mockk

### PR DESCRIPTION
**Pull request by jGleitz**
This is #743 but without the parallel build. Debugging the build logic to figure out what is wrong will take longer, so I think it’s worth getting these changes in first.

______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: logic/atrium-logic-jvm/src/test/kotlin/ch/tutteli/atrium/logic/impl/creating/filesystem/hints/SymbolicLinkResolvingSpec.kt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+//TODO remove file with 1.0.0
+@file:Suppress("DEPRECATION")
```
Comment (robstoll) [inline logic/atrium-logic-jvm/src/test/kotlin/ch/tutteli/atrium/logic/impl/creating/filesystem/hints/SymbolicLinkResolvingSpec.kt:2]: remove

**Review by robstoll — CHANGES_REQUESTED** [review]
Just remove the copy& paste error and it looks good

File: logic/atrium-logic-jvm/src/test/kotlin/ch/tutteli/atrium/logic/impl/creating/filesystem/hints/SymbolicLinkResolvingSpec.kt (hunk 1 above)
Comment (jGleitz) [reply logic/atrium-logic-jvm/src/test/kotlin/ch/tutteli/atrium/logic/impl/creating/filesystem/hints/SymbolicLinkResolvingSpec.kt:2]: :facepalm:

Comment (jGleitz) [reply logic/atrium-logic-jvm/src/test/kotlin/ch/tutteli/atrium/logic/impl/creating/filesystem/hints/SymbolicLinkResolvingSpec.kt]: Done.

**Comment by robstoll** [comment]
@jGleitz do you want to remove mockk in other places as well?

**Review by robstoll — APPROVED** [review]

**Comment by jGleitz** [comment]
>  do you want to remove mockk in other places as well?

I tried. But in one instance, we use `mockk` in a commons project. I saw no easy solution for that, so I left it. Do you have a suggestion?

**Comment by robstoll** [comment]
Unfortunately not no, probably the reason why we stuck with it even though my last knowledge is that it is not further developed

**Pull request merged**